Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/UserDatasetEventSync.java

```diff
@@ -0,0 +1,291 @@
+package org.gusdb.wdk.model.user.dataset.event;
```

Comment:
This file is a mix of new code and code pulled from UserDatasetEventArrayHandler